testsuite/features/secondary/srv_timezone_feature

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2022 SUSE LLC
```

Comment:
The filename is wrong, it must be srv_timezone.feature

Reply:
done good catch